app/assets/stylesheets/custom.scss

```diff
@@ -112,7 +117,7 @@
```

Comment:
Colon after property should be followed by one space<br>`0px` should be written without units as `0`